src/main/resources/config.properties

```diff
@@ -2,4 +2,4 @@
 serializedFilePath = hotelState.ser
 
 # Feature toggles
-hotel.allowChangeStatus = true;
\ No newline at end of file
+hotel.allowChangeStatus = true;
```

Comment:
you can also add here your batabase configs and jut read it. It more correct way